Python/traceback.c

```diff
@@ -1054,6 +1054,27 @@ write_thread_id(int fd, PyThreadState *tstate, int is_current)
     _Py_DumpHexadecimal(fd,
                         tstate->thread_id,
                         sizeof(unsigned long) * 2);
+
+    // Write the thread name
+#ifndef MS_WINDOWS
+    char name[_PYTHREAD_NAME_MAXLEN+1];
+    pthread_t thread = tstate->thread_id;
+#ifdef HAVE_PTHREAD_GETNAME_NP
+    int rc = pthread_getname_np(thread, name, Py_ARRAY_LENGTH(name));
+#else /* defined(HAVE_PTHREAD_GET_NAME_NP) */
+    int rc = 0; /* pthread_get_name_np() returns void */
+    pthread_get_name_np(thread, name, Py_ARRAY_LENGTH(name));
+#endif
+    if (!rc) {
+        size_t len = strlen(name);
+        if (len) {
```

Comment:
```suggestion
    if (rc == 0) {
        size_t len = strlen(name);
        if (len > 0) {
```

Would this be clearer?

Reply:
`if (!rc)` and `if (len)` are commonly used in C. I prefer this style.